mautic/mautic
# [UI] Modals

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13164 



## Description

Specifically:
- Makes overlays blurred when a modal is open
- Integrates with #13752 to ensure fields keep visible while on top of the modal background




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/7296bc16-7680-48f1-a7e7-45eb8e798846)                                       |![image](https://github.com/user-attachments/assets/025aea5b-5755-4788-9f33-23247cf52a37)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Categories > New > See white fields and blurred background
